### DISCUSSION TITLE
Multiple users interacting with the same patch

### DISCUSSION THREAD

**Discussion by GiulianoComoglio** [sebpiq/WebPd#145] [Q&A]
Dear WebPd developers, I am Giuliano Comoglio, composer and master composition student at Codarts, Rotterdam. I've created this discussion hoping it could be more broadly helpful.

I'm currently conducting an artistic research based on audience interactivity with live electornics. I've been using pure data for many years and just discovered WebPd - it's really awsome. I was wondering if there is any way to make an entire audience connect and interact with the very same patch - I would like the patch to receive and distinguish between multiple user inputs.
The project would involve an audio input, captivating one or more instruments on stage, and the audience using the WebPd GUI to maninpulate this input based on messages buttons, sliders or toggles triggering sound manipulations. 
It would be even better to have some ways of distinguishing between each of the audience members connected to the session, and to send them individual settings and options as the performance proceeds.

Right now I have no knowledge about web development languages, but I'm very open to start learning those if there might be any solution involving them.

Thank you in advance very much!

**Comment by sebpiq** [comment]
Hi @julenComposer !

I suggest you ask your question on the Pure Data discord (channel webpd) : https://discord.com/invite/AZ43djV

It is more active than here, with more users reading and answering ;)

**Comment by GiulianoComoglio** [comment]
Dear @sebpiq, thank you so much for answering me and for the indication! I'll go there then!
